programs/drift/src/state/order_params.rs

```diff
@@ -216,12 +228,16 @@ impl OrderParams {
         }
         // only update auction start price if the contract tier isn't Isolated
         if perp_market.amm.oracle_source != OracleSource::Prelaunch {
-            let new_start_price_offset =
-                OrderParams::get_perp_baseline_start_price_offset(perp_market, self.direction)?;
+            let (new_start_price_offset, new_end_price_offset) =
+                OrderParams::get_perp_baseline_start_end_price_offset(
+                    perp_market,
+                    self.direction,
+                    2,
```

Comment:
why 2?

Reply:
to start, wanted to only filter extremes like in `derive_market_order_auction_params` 

default is 1